src/main/java/chadbod/TaskList.java

```diff
@@ -0,0 +1,92 @@
+package chadbod;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks in the ChadBod application.
+ */
+public class TaskList {
+
+    private static final int TASKLIST_DISPLAY_OFFSET = 1;
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructor for TaskList class, initializes an empty task list.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Get the number of tasks in the task list.
+     *
+     * @return The number of tasks in the list.
+     */
+    public int getTaskCount() {
+        return tasks.size();
+    }
+
+    /**
+     * Get a task at a specific index in the task list.
+     *
+     * @param i The index of the task to retrieve.
+     * @return The task at the specified index.
+     */
+    public Task getTask(int i) {
```

Comment:
Using `i` as the parameter name might not communicate to users what input they should be placing into it. An alternative would be to use `index`